docs/computing/jobs/slurm-quickstart.md

```diff
@@ -10,6 +10,42 @@ through the submission of jobs by the user. A job describes the computing
 resources required to run application(s) and how to run it. LUMI uses Slurm
 as the batch scheduler and resource manager.
 
+
+In order to run, you need a project allocation. 
+You need to specify your project ID in your job script (or via the command line
+when submitting your job) in order for your job to be submitted to the queue. 
+
+!!! missing
+
+    Commands to gather information about the project and quota are not
+    available yet. However, you can use the `groups` command to retrieve your 
+    project ID when connected to LUMI: you should see that you are part of a 
+    group named `project_xxxxxxxxx`.
+
+
+
+Here is a typical batch script for Slurm. This script runs an application
+on 2 compute nodes with 16 MPI ranks on each node (32 total) and 8 OpenMP 
+threads per rank.
+
+```
+$ cat batch_script.slurm
+#!/bin/bash -l
+#SBATCH --job-name=test-job
```

Comment:
I wouldn't maybe use a dash "-" in a job name, but it's maybe just my own practice. (Not sure if it can even potentially mess up anything there.)

Reply:
OK. I hope whoever wrote it in the first place tested it. I'll leave it for now.